lib/src/base-styles/_nx-tables.scss

```diff
@@ -119,10 +119,16 @@
   position: relative;
   z-index: 0;
 
-  .nx-cell--header {
+  > thead tr:nth-child(1) th {
     position: sticky;
     top: 0;
-    z-index: 1;
+    z-index: 1
+  }
+
+  > thead tr:nth-child(2) th {
+    position: sticky;
+    top: 45px;
```

Comment:
45 doesn't seem to be the correct amount; the row still shifts when scrolling.  The header cells are 48px high so you presumably need this to be 48.  You should extract a scss variable for that height to keep those two values in sync.

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/a954210b17845f9ed60e40fd1bf15de1f5723c9a